src/OpenLoco/GameCommands/GameCommands.cpp

```diff
@@ -184,6 +185,29 @@ namespace OpenLoco::GameCommands
             return loc_4313C6(esi, regs);
         }
 
+        auto isGhost = (flags & Flags::flag_6) != 0;
+        if (!isGhost && Network::isConnected())
+        {
```

Comment:
do we know how the original handled this?

Reply:
I don't know. I don't recall ever seeing ghosts years ago when I played the original multiplayer.